### ISSUE TITLE
How to generate pngs automatically from the svg files?

### ISSUE THREAD

**Issue by PHPeon** [linuxmint/mint-y-icons#376]
How do i automatically convert the svgs which i submit into png icons?

**Comment by CZmisaCZ** [comment]
@PHPeon same issue, did you figured out how convert svgs?

**Comment by PHPeon** [comment]
> @PHPeon same issue, did you figured out how convert svgs?

No, not really sir Webster or Clement always converted them for me through i process which i do not fully understand

**Comment by CZmisaCZ** [comment]
For anyone reading this, you can render pngs out of svgs running: `render-apps-categories-bitmaps.py` that is located in `src` of this project. 

Alternatively you can run:

```bash
cd src
python3 ./render-apps-categories-bitmaps.py NAME-OF-YOUR-ICON
```
To re-render your icon.
